All tests passed + main DONE

Ivosel opened the pull request.

Ivosel committed:
All tests passed + main DONE

ncaklovic commented:
Realno, ne treba vam ni jedan _member_ u klasama za životinje (makar svi bili _const_) - zauzimaju vam memoriju po svakom objektu, a mogli ste direktno u funkcijama vratiti konstantu (6 ili "sparrow"). 
Pogotovo vam ne trebaju `id` kao privatni _const_ _memberi_ - koji se nigdje ne koriste (mogli bi biti _public static const_ _memberi_, to bi imalo smisla).

Varam li se ili ste u `main`-u dva puta napravili istu životinju? Pa `add_animal()` vam vraća string, niste morali još jednom raditi istu životinju da biste ispisali naziv vrste.

Postavljate `id` na 1 pa za uvjet petlje imate `id`, kad već imate `break` u petlji bolje je pisati:

```
while (true){
	uint32_t id; cin >> id;
	if (id == 0) 
		break;
	// ...
}
```

Ivosel committed:
- Remove unnecesarry data members
- Main only calls 1 add_animal() function
- While loop sorted without preinitialisation of id

ncaklovic commented:
Sve u redu.
